data/src/test/java/co/nimblehq/chorn/survey/data/test/MockUtil.kt

```diff
@@ -20,13 +20,23 @@ object MockUtil {
             every { httpException.code() } returns response.code()
             every { httpException.message() } returns response.message()
             every { httpException.response() } returns response
-            every { responseBody.string() } returns "{\n" +
-                    "  \"message\": \"message\"\n" +
-                    "}"
+            every { responseBody.string() } returns """
+{
+  "errors": [
+    {
+      "source": "errorSource",
+      "detail": "errorDetail",
+      "code": "errorCode"
+    }
+  ]
+}
+"""
```

Comment:
Please reformat to correct the json format

```suggestion
            every { responseBody.string() } returns "{\n" +
                    "  \"errors\": [\n" +
                    "    {\n" +
                    "      \"source\": \"errorSource\",\n" +
                    "      \"detail\": \"errorDetail\",\n" +
                    "      \"code\": \"errorCode\"\n" +
                    "    }\n" +
                    "  ]\n" +
                    "}"
```